### ISSUE TITLE
Pytorch

### ISSUE THREAD

**Issue by AlexNmSED** [junyuchen245/Unsuprevised_Seg_via_CNN#7]
你好，是否有Pytorch的训练代码呢？

**Comment by junyuchen245** [comment]
Hi @AlexNmSED 

Unfortunately, I don't have a pytorch implementation of this code. However, the main contribution of this work is the unsupervised [loss function](https://github.com/junyuchen245/Unsuprevised_Seg_via_CNN/blob/master/Unsupervised_Seg/losses.py), which should be trivial to adapt to pytorch.

Alternatively, you are welcome to check out our more recent work on [multi-class unsupervised segmentation loss](https://github.com/junyuchen245/Semi-supervised_FCM_Loss_for_Segmentation), which was implemented in both Python and Tensorflow.

Junyu
